src/main/resources/application.properties

```
@@ -1 +1,7 @@
+spring.datasource.url=jdbc:h2:mem:productdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=root
+spring.datasource.password=pikador10
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
 
+spring.h2.console.enabled=true
```

we have to enable generated SQL statement logging
```suggestion
spring.h2.console.enabled=true

logging.level.org.hibernate.SQL=DEBUG
logging.level.org.hibernate.type.descriptor.sql.BasicBinder=TRACE
```

---

TRACE може бути redundant в проді